pkgs/games/openmw/default.nix

```diff
@@ -18,37 +18,14 @@
 , lz4
 , recastnavigation
 , VideoDecodeAcceleration
+, libyamlcpp
+, luajit
 }:
 
 let
-  openscenegraph_openmw = (openscenegraph.override { colladaSupport = true; })
-    .overrideDerivation (self: {
-      src = fetchFromGitHub {
-        owner = "OpenMW";
-        repo = "osg";
-        rev = "bbe61c3bc510a4f5bb4aea21cce506519c2d24e6";
-        sha256 = "sha256-t3smLqstp7wWfi9HXJoBCek+3acqt/ySBYF8RJOG6Mo=";
-      };
-      patches = [
-        (fetchpatch {
-          # For Darwin, OSG doesn't build some plugins as they're redundant with QuickTime.
-          # OpenMW doesn't like this, and expects them to be there. Apply their patch for it.
-          name = "darwin-osg-plugins-fix.patch";
-          url = "https://gitlab.com/OpenMW/openmw-dep/-/raw/0abe3c9c3858211028d881d7706813d606335f72/macos/osg.patch";
-          sha256 = "sha256-/CLRZofZHot8juH78VG1/qhTHPhy5DoPMN+oH8hC58U=";
-        })
-      ];
-    });
```

Comment:
Are you sure that openmw doesn't still recommend using their fork?

Reply:
Yes, it is [recommended](https://wiki.openmw.org/index.php?title=Development_Environment_Setup#OpenSceneGraph) and we should keep using it.

Reply:
As I mentioned in the description (though not very prominently) the [diff to upstream](https://github.com/OpenMW/osg/compare/3.6...openscenegraph:OpenSceneGraph:OpenSceneGraph-3.6) is virtually nonexistent.